api/Document.json

```
@@ -8624,7 +8629,7 @@
             },
             "edge": [
               {
-                "version_added": "12"
+                "version_added": "79"
```

```suggestion
                "version_added": "12"
```

Pointer Events was proposed and shipped first by Microsoft Edge. 
```suggestion
                "version_added": "79"
```